ert_data/measured.py

```diff
@@ -1,77 +1,124 @@
 from collections import defaultdict
-
+from typing import Optional, List
+import deprecation
 import pandas as pd
 import numpy as np
 
 from ert_data import loader
+from ert_shared.libres_facade import LibresFacade
+
+# importlib.metadata added in python 3.8
+try:
+    from importlib import metadata
+
+    __version__ = metadata.version("ert")
+except ImportError:
+    from pkg_resources import get_distribution
+
+    __version__ = get_distribution("ert").version
+
+
+class MeasuredData(pd.DataFrame):
```

Comment:
Not completely sure if I see main benefits of having this derived from DataFrame. Are we planning to use / override other functionalities of DataFrame like `merge` (joining different measured data) or `corr` (computing the correlation among observations)?

Reply:
The way it is before this, is that there is a property, `data`, which has a dataframe. You can interact with it, but it is not quite pandas-like. 

As I see it we would not like to override DataFrame functionalities, but make them easily available to the user. Then we can expand it with some functionality that is specific to our cause (like filtering on ensemble mean for example).